evm/src/main/java/org/hyperledger/besu/evm/precompile/MainnetPrecompiledContracts.java

```diff
@@ -86,4 +88,11 @@ public static void populateForBLS12(
     registry.put(Address.BLS12_MAP_FP_TO_G1, new BLS12MapFpToG1PrecompiledContract());
     registry.put(Address.BLS12_MAP_FP2_TO_G2, new BLS12MapFp2ToG2PrecompiledContract());
   }
+
+  public static void populateForCancun(
+      final PrecompileContractRegistry registry, final GasCalculator gasCalculator) {
+    populateForIstanbul(registry, gasCalculator);
```

Comment:
Why is this not populateForBLS12?

Reply:
Only because shipping fork for BLS12 is still tbd, so didn't want to build on top of that.